### PR TITLE
OCPBUGS-7267: add SeccompProfile to Pod and Container accessors/mutators

### PR DESCRIPTION
**Pull request by stlaz**
/assign @openshift/openshift-team-auth 
/cc @deads2k

### CODE REVIEW

**Comment by stlaz** [comment]
needed for https://github.com/openshift/apiserver-library-go/pull/100

File: pkg/securitycontext/accessors.go [hunk 1]
```diff
@@ -171,6 +173,19 @@ func (w *podSecurityContextWrapper) SetRunAsNonRoot(v *bool) {
 	w.ensurePodSC()
 	w.podSC.RunAsNonRoot = v
 }
+func (w *podSecurityContextWrapper) SeccompProfile() *api.SeccompProfile {
```
Comment (deads2k) [inline pkg/securitycontext/accessors.go:176]: add these to a `patch_accessors.go` file

File: pkg/securitycontext/accessors.go [hunk 2]
```diff
@@ -46,6 +47,7 @@ type PodSecurityContextMutator interface {
 	SetRunAsUser(*int64)
 	SetRunAsGroup(*int64)
 	SetRunAsNonRoot(*bool)
+	SetSeccompProfile(*api.SeccompProfile)
```
Comment (deads2k) [inline pkg/securitycontext/accessors.go:50]: create new interface you can cast to that has this method and put it in `patch_accessors.go` to reduce conflicts.

**Comment by deads2k** [comment]
is there a part 2?  what uses this?

**Comment by deads2k** [comment]
/lgtm
/approve

If this ends up turning into a carry, see my comments on it.  This fixes a security problem, merging.

**Pull request merged**